server/grpc.go

```diff
@@ -75,32 +79,32 @@ type StreamServer struct {
 	removeClients  chan manager.Client
 	payloads       chan event.Payload
 	resfreshEvents chan refreshEvents
-	errChan        chan error
 	forceCloseChan chan manager.Client
 	pubsub         pubsub.PubSuber
 	accessLogger   *zap.Logger
 	errorLogger    *zap.Logger
 }
 
-func NewStreamServer(opt Option) *StreamServer {
+func NewStreamServer(opt Option) (*StreamServer, error) {
```

Comment:
__[golint]__
* [golint] exported function NewStreamServer should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->